### PR TITLE
Add 1.11.1/1.11.2 protocol

### PR DESCRIPTION
**Pull request by mathiascode**
No new features, but 1.11.1 and 1.11.2 clients can join.

### CODE REVIEW

File: src/Protocol/Protocol_1_11.cpp [hunk 1]
```diff
@@ -187,3 +187,51 @@ void cProtocol_1_11_0::HandlePacketStatusRequest(cByteBuffer & a_ByteBuffer)
 
 
 
+cProtocol_1_11_1::cProtocol_1_11_1(cClientHandle * a_Client, const AString &a_ServerAddress, UInt16 a_ServerPort, UInt32 a_State) :
```
Comment (madmaxoft) [inline src/Protocol/Protocol_1_11.cpp]: Style: spaces around `&`

**Comment by madmaxoft** [comment]
Looks alright, after the style issue is fixed, feel free to merge.

File: src/Protocol/Protocol_1_11.cpp (hunk 1 above)
Comment (falconxunit) [inline src/Protocol/Protocol_1_11.cpp]: ok good show me

**Comment by madmaxoft** [comment]
@falconxunit what are you on about? It's the second time you're babbling nonsense in our tracker.

**Comment by mathiascode** [comment]
Done. Some packet IDs and code was from the 1.8 protocol, which caused client disconnection when mobs spawned or the player tried to collect pickups. I've corrected these.

Every mob works properly now, except for the skeleton, which is invisible. I guess we can fix this later.

**Comment by madmaxoft** [comment]
Does this mean we have a more-or-less functional 1.11 support? Good job!

**Comment by Pokechu22** [comment]
> Every mob works properly now, except for the skeleton, which is invisible. I guess we can fix this later.

Do wither skeletons work?  Also, do zombie villagers work?  Finally, do the horse variants work?  Since the IDs of these entities were split in 1.11, I'm not sure if they'd be functional.

Still, good work!

**Comment by madmaxoft** [comment]
Let's not get carried away in this PR. It is complete now, anything else is a good candidate for another PR. Making further changes would have made it need a new review, much more in-depth.

**Comment by mathiascode** [comment]
Wither skeletons, zombie villagers, zombie horses and skeleton horses don't spawn at the moment. Further works needs to be done to make these functional again.

**Comment by Pokechu22** [comment]
Yep, I expected that (and it's perfectly reasonable).  Just wanted to make sure that _all_ of them were broken, not just one of them (which would be very strange).

**Pull request merged**